src/egon/data/datasets/gas_grid.py

```diff
@@ -18,6 +19,16 @@
 from geoalchemy2.shape import from_shape
 from pathlib import Path
 
+#class GasGrid(Dataset):
+#     def __init__(self, dependencies):
+#         super().__init__(
+#             name="GasGrid",
+#             version="0.0.0.dev",
+#             dependencies=dependencies,
+#             tasks=(etrago_input_data, download_data_bundle, osmtgmod_pypsa),
+#         )

```

Comment:
I'm wondering why these lines are there but commented... Do you wanted to create a `Dataset`-class for the GasGrid?

Reply:
I thought about it, but I ran into errors and because it was not the purpose of the branch, I decide to let it as it is.